electron-app/src/components/settings/ExchangeSettings.vue

```diff
@@ -37,6 +37,17 @@
             :disabled="isConnected"
             @click:append="showSecret = !showSecret"
           ></v-text-field>
+          <v-text-field
+            v-if="selectedExchange === 'coinbasepro'"
+            v-model="passphrase"
+            class="exchange-settings__fields__passphrase"
+            prepend-icon="fa-key"
+            :append-icon="showKey ? 'fa-eye' : 'fa-eye-slash'"
+            label="Passphrase"
+            :disabled="isConnected"
+            :type="showKey ? 'text' : 'password'"
```

Comment:
You need a new variable here instead of show key, otherwise this field will show/hide the password together with the key input.